fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/version/VersionHistoryService.java

```diff
@@ -126,7 +126,7 @@ public void insertVersionHistoriesInTx(Collection<TypeNameVersion> versionHistor
      * @param version
      */
     public void insertVersionHistoryInTx(String objectSchema, String objectType, String objectName, int version) {
-        AddVersionDAO dao = new AddVersionDAO(adminSchemaName, objectSchema, objectType, objectName, version);
+        AddVersionDAO dao = new AddVersionDAO(adminSchemaName, objectSchema.toUpperCase(), objectType, objectName, version);
```

Comment:
Going forward this would always ensure the schemaname is uppercase in the VHS table